frontend/src/views/web/partials/sidebar.html

```diff
@@ -47,53 +47,37 @@
                 </li>
             </ul>
         </div>
-    </div>
-</div>
-<!-- define all main links here for navigation -->
-<div class="ev-sidebar ev-fixed z-depth-3 fixed hide-on-med-and-down">
-    <div class="side-intro ev-md-container fixed">
-        <!-- <scroll-shadow></scroll-shadow> -->
-        <div class="w-400 side-logo"><a class="white-text" ui-sref="home" ui-sref-active="active">EvalAI</a></div>
-        <div class="w-300 text-light-gray">Welcome {{web.name}}!</div>
-    </div>
+    </div> -->
     <div class="links-section-outer">
-        <div class="links-section ">
-            <ul class="nav-links">
-                <li>
-                    <a class="waves-effect waves-dark text-light-gray ev-md-container" ui-sref="web.dashboard" ui-sref-active="side-active-link">
-                        <i class="fa fa-signal side-link-icon"></i> Dashboard</a>
-                </li>
-                <li>
-                    <a class="waves-effect waves-dark text-light-gray ev-md-container" ui-sref="web.challenge-main" ui-sref-active="side-active-link">
-                        <i class="fa fa-fire side-link-icon"></i> All Challenges</a>
-                </li>
-                <li>
-                    <a class="waves-effect waves-dark text-light-gray ev-md-container" ui-sref="web.teams" ui-sref-active="side-active-link">
-                        <i class="fa fa-users side-link-icon"></i> My Participant Teams</a>
-                </li>
-
-                <li>
-                    <a class="waves-effect waves-dark text-light-gray ev-md-container" ui-sref="web.challenge-host-teams" ui-sref-active="side-active-link">
-                        <i class="fa fa-paper-plane side-link-icon"></i>My Host Teams</a>
-                </li>
-            </ul>
+        <div class="links-section">
             <ul>
-                <li class="text-highlight side-title w-500 ">My Details</li>
                 <li>
-                    <a class="waves-effect waves-dark text-light-gray ev-md-container" ui-sref="web.profile" ui-sref-active="side-active-link">
-                        <i class="fa fa-user side-link-icon"></i> My Profile</a>
+                    <a class=" text-light-black" ui-sref="web.dashboard" ui-sref-active="side-active-link">
+                        <div>
```

Comment:
Yes, it will be better to use css instead of semantic tags but I did that intentionally because I don't want to change the default icons styles here. 
But let me think again, will it be good to use div here or not.